data-plane/dispatcher/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>data-plane</artifactId>
+    <groupId>dev.knative.eventingkafkabroker</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>dispatcher</artifactId>
+
+  <build>
```

Comment:
If we use jib, you don't need to configure the maven-shade-plugin. In case you wanna run it locally to do some testing, just use https://reactiverse.io/vertx-maven-plugin/